specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2018-07-10/service.json

```diff
@@ -14702,6 +14759,14 @@
           "items": {
             "$ref": "#/definitions/RecoveryPlanGroup"
           }
+        },
+        "providerSpecificDetails": {
```

Comment:
Is this readonly?

If not, this would be considered a breaking change and require a new api version: See 'New property added to response'
https://microsoft.sharepoint.com/:w:/r/teams/azureresourcemanagerteam/_layouts/15/guestaccess.aspx?e=xEcppo&share=EWXsAQ1yx25KkyYCeeWGUgwBSBxEdUDEbHi6FZ__U8EOQw

Reply:
Updated with read only